nengo_viz/disposable_js.py

```diff
@@ -35,6 +35,12 @@ def configmodal():
     js.append(show_modal_js())
     return '\n'.join(js)
 
+def helpmodal():
+    js = [add_modal_title_js('Hotkeys list')]
```

Comment:
This function isn't actually using any information from the Python server, so there's no need to send work back to Python. Just do this all client-side (i.e., in Javascript).